<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>browseWidget</class>
 <widget class="QWidget" name="browseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>529</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Artist:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="artistList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerItem</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Genre:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="genreList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerItem</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,5,0,1">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Track:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="trackList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Added ...</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="addContBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Add &amp;
 C&amp;ontinue</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addCloseBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Add &amp;&amp;
 Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>browseWidget</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchBtn</sender>
   <signal>clicked()</signal>
   <receiver>browseWidget</receiver>
   <slot>switchToSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addCloseBtn</sender>
   <signal>clicked()</signal>
   <receiver>browseWidget</receiver>
   <slot>addClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addContBtn</sender>
   <signal>clicked()</signal>
   <receiver>browseWidget</receiver>
   <slot>addContinue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>artistList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>browseWidget</receiver>
   <slot>onArtistChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>genreList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>browseWidget</receiver>
   <slot>onGenreChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cancel()</slot>
  <slot>switchToSearch()</slot>
  <slot>addClose()</slot>
  <slot>addContinue()</slot>
  <slot>onArtistChanged()</slot>
  <slot>onGenreChanged()</slot>
 </slots>
</ui>
